bundles/org.openhab.binding.mqtt.homeassistant/pom.xml

```diff
@@ -15,7 +15,7 @@
   <name>openHAB Add-ons :: Bundles :: MQTT HomeAssistant Convention</name>
 
   <properties>
-    <graalpy.version>25.0.0</graalpy.version>
+    <graalpy.version>25.0.1</graalpy.version>
```

Comment:
you can use "${graalpy.version}" here

Reply:
ahh, sorry my mistake, I want to say, that you must not define it here. There is already an approach of having a more global variable

https://github.com/openhab/openhab-addons/blob/1aca2fbc444cca0271a11e04bf4ce40f10f71afb/pom.xml#L78

Reply:
is changed as part of my current pull request

https://github.com/openhab/openhab-addons/pull/19568/commits/f799ff50a20515e5d57565d533e90261a742dc62